packages/cli/src/commands/stop.js

```diff
@@ -1,6 +1,8 @@
 const pify = require('pify')
 const pm2 = require('pm2')
 
+const { SEEDER_DAEMON } = require('./start')
```

Comment:
Looks good to me, if more variables are needed in the future we can come up with a new `constants.js` file or something like that.